### PR TITLE
[AQM] Implement output capabilities for production and bring pm25 as a 3D diagnostic tracer in UFSWM

### LINKED ISSUE #21: Missing operational products
Air quality operational products, such as hourly max and average surface concentrations of ozone and PM2.5, need to be output by the model.

### PR DESCRIPTION
**Pull request by BrianCurtis-NOAA**
# PR Checklist

- [X] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [X] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR
are specified below.
- [X] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [X] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Description

Bring output capabilities from AQM for PM2.5 and use PM2.5 as 3D tracer in UFSWM

Tested on Hera/Orion, needs new baseline for JUST regional_atmaq test.

### Issue(s) addressed
- fixes noaa-emc/aqm/issues/21

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel
- [x] cheyenne.gnu
- [x] gaea.intel
- [x] jet.intel
- [x] wcoss2.intel
- [ ] acorn.intel
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies
- waiting on noaa-emc/aqm/pull/37

### CODE REVIEW

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: hera
Compiler: gnu
Job: RT
[RT] Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/1125089145/20221117230017/ufs-weather-model
[RT] Error: Test cpld_control_p8 049 failed in run_test failed 
Please make changes and add the following label back: hera-gnu-RT

**Comment by jkbk2004** [comment]
@BrianCurtis-NOAA all RT tests are done. we can start merging process.

File: tests/parm/aqm/aqm.rc [hunk 1]
```diff
@@ -29,7 +29,7 @@ ctm_pmdiag: true
 #
 #  Input emissions
 #
-emission_sources: anthro fire
+emission_sources: anthro fire prod
```
Comment (rmontuoro) [inline tests/parm/aqm/aqm.rc:32]: fengsha needs to be added to the emission sources:
```
emission_sources: anthro fire fengsha prod
```

**Review by jkbk2004 — APPROVED** [review]

**Review by zach1221 — APPROVED** [review]

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: orion
Compiler: intel
Job: RT
[RT] Repo location: /work/noaa/nems/emc.nemspara/autort/pr/1125089145/20221117144518/ufs-weather-model
Please make changes and add the following label back: orion-intel-RT

**Review by rmontuoro — CHANGES_REQUESTED** [review]
The AQM configuration for the regional_atmaq regression test needs to be updated as per provided comment.

File: tests/parm/aqm/aqm.rc (hunk 1 above)
Comment (BrianCurtis-NOAA) [reply tests/parm/aqm/aqm.rc:32]: Fengsha is currently breaking the RT, but I know there are fixes coming in immediately. This can be addressed with that.

Comment (BrianCurtis-NOAA) [reply tests/parm/aqm/aqm.rc:32]: see https://github.com/NOAA-EMC/AQM/issues/36

**Pull request merged**